src/WorldStorage/NBTChunkSerializer.cpp

```diff
@@ -378,6 +380,18 @@ class SerializerCollector:
 
 
 
+	void AddBannerEntity(cBannerEntity * a_Entity)
+	{
+		mWriter.BeginCompound("");
+			AddBasicTileEntity(a_Entity,"Banner");
+			mWriter.AddInt("Base", static_cast<int>(a_Entity->GetBaseColor()));
```

Comment:
The wiki lists a `Patterns` list which includes the base colour I believe

Reply:
Ah no, this is incorrect. The "Base" in Patterns refers to the Fully Field design.

Nonetheless, am I right in saying the banner's wool colour is encoded in the block, or in older versions the block meta?

Reply:
hmmm... if I recall correctly I checked the entities tags in the notchian implementation with the `/data` command and there was a base tag... and yeah the block or the meta should determine the color

Reply:
Oh and the meta stores the rotation